### PR TITLE
Move to spring-cloud-dataflow-build

### PR DESCRIPTION
**Pull request by jvalkeal**
- Simple change to use spring-cloud-dataflow-build instead
  of spring-cloud-build.
- Relates spring-cloud/spring-cloud-dataflow#3670

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,8 +12,8 @@
 
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
-		<artifactId>spring-cloud-build</artifactId>
-		<version>2.2.0.RELEASE</version>
+		<artifactId>spring-cloud-dataflow-build</artifactId>
```
Comment (ilayaperumalg) [inline pom.xml]: It looks weird to have the parent artifact Id being `spring-cloud-dataflow-build` for the deployers. Can we come up with any other name that doesn't have `dataflow` in it? Perhaps, we can create `spring-cloud-deployer-build` that acts as a parent for all the deployers?

**Comment by ilayaperumalg** [comment]
I guess we need to create deployer master to be in `2.2.x` across the board for these changes.

File: pom.xml (hunk 1 above)
Comment (jvalkeal) [reply pom.xml]: There's going to exactly one parent and there's not good naming for dataflow/skipper/deployers other than they all belong to "dataflow" family.

**Comment by jvalkeal** [comment]
I'm not sure we need to bump deployer versions for this.

**Comment by ilayaperumalg** [comment]
> I'm not sure we need to bump deployer versions for this.

Because the SCDF/Skipper 2.3.x/2.2.x use the same versions of the deployers

**Comment by jvalkeal** [comment]
Right, that's a good point. Completely forget that might become an issue.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Rebased and merged as d89354f

**Pull request closed without merging**